x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/summary_panel/datasets_activity.tsx

```diff
@@ -26,7 +25,7 @@ export function DatasetsActivity() {
       tooltip={summaryPanelDatasetsActivityTooltipText}
       value={text}
       isLoading={isDatasetsActivityLoading}
-      isUserAuthorizedForDataset={isUserAuthorizedForDataset}
+      isUserAuthorizedForDataset={true}
```

Comment:
`Last Activity` is now always enabled through dataStreams API `verbose` mode

Reply:
Can we completely remove this prop in favour of this change now ?

Reply:
This generic component is used also for `Estimated Size`. In https://github.com/elastic/logs-dev/issues/179 I'll do a refactor